solutions/microfrontends/packages/acme-pages/package.json

```diff
@@ -28,24 +31,35 @@
       ]
     }
   },
-  "files": [
-    "src"
-  ],
+  "publishConfig": {
+    "typesVersions": {
+      "*": {
+        "*": [
+          "dist/*.d.ts",
+          "dist/*/index.d.ts"
+        ]
+      }
+    }
+  },
   "peerDependencies": {
-    "next": "^12.1.0",
-    "react": "^17.0.0",
-    "react-dom": "^17.0.0"
+    "next": "*",
+    "react": "^17.0.0 || ^18.0.0-0",
```

Comment:
Do we still need 17?

Reply:
It's okay for the purposes of the example, which is showing how to have shared packages, and these might allow a certain version to be used, even if 17 is not used in the example.